ReactWindows/ReactNative.Net46/Tracing/Tracer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace ReactNative.Tracing
+{
+    class NullDisposable : IDisposable
+    {
+        public void Dispose()
+        {
+
+        }
+    }
+
+    public class NullLoggingActivityBuilder
```

Comment:
As such, you'll have to either add docs to the public surfaces you've added here, or just remove `public` from Tracer and NullLoggingActivityBuilder 

---
In reply to: [82050122](https://github.com/ReactWindows/react-native-windows/pull/740#discussion_r82050122) [](ancestors = 82050122)